Terminal.Gui/Drawing/Cell.cs

```diff
@@ -23,7 +23,7 @@ public class Cell {
 	///// <remarks>
 	///// Only valid in the rare case where <see cref="Rune"/> is a combining sequence that could not be normalized to a single Rune.
 	///// </remarks>
-	//internal Rune CombiningMark { get; set; }
+	internal Rune CombiningMark { get; set; }
```

Comment:
I believe you were correct in asserting this should be `CombiningMarks` and be a list:

![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/c76de708-84cf-4f1d-aab9-d8c60a773a96)

Reply:
In this case no. It's only needed do add to the `CombiningMarks` if the `normalized` string length on the `ConsoleDriver` is grater than 1, otherwise is 0.

Reply:
I do not waste more time with this. For sure it's a limitation of the terminal. If you can put it working then you'll deserve a medal :-)

Reply:
However, I think I was partially right: the .NET Console API does not properly pass combining marks on. Thus combining marks will not work on `NetDriver`.

Reply:
WindowsDriver will need signficant refactoring to utilize `Cell.CombingMarks`. 

a) It will never work with them for `force16colors` because `WriteConsoleOutput` only supports a `char` per buffer position.
b) `WriteConsole` may or may not work like `Console.Write`. Regardless, `WindowsConsole.ExtendedCharInfo` needs to be expanded to hold the combining marks. Might as well just nuke that data structure and replace it with `Cell`, no?

Reply:
The `StringBuilder` handles correctly but the `System.Console` doesn't.

Reply:
> WindowsDriver will need signficant refactoring to utilize `Cell.CombingMarks`.
> 
> a) It will never work with them for `force16colors` because `WriteConsoleOutput` only supports a `char` per buffer position.

It's possible to put surrogate pair into two consecutive char position and the `WriteConsoleOutput` will render well. The problem is the dealignment caused on the right. I didn't yet found a solution for that but I already can write surrogate pairs on `WindowsDriver`, instead of showing the rune replacements.

> b) `WriteConsole` may or may not work like `Console.Write`. Regardless, `WindowsConsole.ExtendedCharInfo` needs to be expanded to hold the combining marks. Might as well just nuke that data structure and replace it with `Cell`, no?

I really didn't look well into this yet because it deal with Ansi escape sequences which is only supported by the Windows Terminal.